### PR TITLE
add my work

### PR DESCRIPTION
**Pull request by xiaxia330**
please help @akblal @Louisaflor @noattongi

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ app.use(express.static('client/dist'));
 
 
 
-app.listen(3000,()=>{
-  console.log('listening on port 3000');
+app.listen(4000,()=>{
```
Comment (noattongi) [inline server/index.js]: could you make sure to change this back to port 3000

Comment (xiaxia330) [reply server/index.js]: ok

**Review by Louisaflor — APPROVED** [review]
looks good

**Review by akblal — APPROVED** [review]
approved

**Pull request merged**